src/components/Container/index.tsx

```diff
@@ -7,5 +7,4 @@ export const Container = styled.div`
   margin-right: auto;
   max-width: ${MEDIA_WIDTHS.upToMedium}px;
   width: 100%;
-  padding: 0 1rem;
```

Comment:
Try this on mobile to see why padding is crucial to add some buffer zone between page and actual page content.

Reply:
we already have outer wrapper try it on mobile it should have space...